docs/api/mount.md

```diff
@@ -40,7 +40,7 @@ describe('<Foo />', () => {
 });
 ```
 
-## `mount(node[, options]) => ReactWrapper`
+## `mount(node,[options]) => ReactWrapper`
```

Comment:
this is not a typo. the square brackets denote "optional", and if you remove the square brackets and everything in between them, the comma shouldn't still be there.